### ISSUE TITLE
[Accessibility][Product Teaser] The product image has an insufficient alternative text attribute

### ISSUE THREAD

**Issue by LevelAccessA11y** [adobe/aem-core-wcm-components#1768]
**Steps to Reproduce**
1. With NVDA running, navigate to product teaser image using the down arrow key.

**Current Behavior**
The alternative text value for the image is /content/dam/core-components-examples/library/cif-sample-assets/catalog/product/summit-watch/mg03-br-0.jpg, which does not provide users of screen readers an informative description of the image.

**Expected behavior/code**
All meaningful images have informative, concise alternative text.

**Environment**
Chrome 91, Windows 10, NVDA 2020.4 
AEM Core Components v. 2.17  

**Affected Users**
Screen reader users, users with low vision

**WCAG SC**
1.1.1 Non-Text Content

**Comment by richardhand** [comment]
@adobe export issue to Jira project SITES
